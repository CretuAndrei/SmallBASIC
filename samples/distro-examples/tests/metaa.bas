INCLUDE "metac.bas"
? "Module: Meta-A"

