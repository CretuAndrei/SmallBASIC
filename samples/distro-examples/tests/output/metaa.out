Module: Meta-B
Module: Meta-A
